### PR TITLE
Add company number when company name is set. And update tests

### PR DESCRIPTION
**Pull request by wdhda-ch**

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: test/routers/handlers/check_details/check.details.unit.ts [hunk 1]
```diff
@@ -262,7 +262,7 @@ describe("check details tests", () => {
             "No"
         );
         expect(resp.text).not.toContain(
-            exampleNonRegisteredBusinessDetails.companyName
+            examplePresenterAccountDetailsInternal.companyName + " (" + examplePresenterAccountDetailsInternal.companyNumber + ")"
```
Comment (ch-skathirvel) [inline test/routers/handlers/check_details/check.details.unit.ts:265]: Instead "", can we concat using `

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by ch-skathirvel — APPROVED** [review]

**Pull request merged**